package.json

```diff
@@ -5,16 +5,18 @@
   "homepage": "http://passmarked.com",
   "preferGlobal": true,
   "engines": {
-    "node": ">=0.12.0"
+    "node": ">=0.12"
   },
   "bin": {
     "passmarked": "index"
   },
   "scripts": {
-    "test": "mocha"
+    "test": "mocha",
+    "coverage": "istanbul cover _mocha --report lcovonly -- test -R spec"
   },
   "author": "Passmarked <info@passmarked.com>",
   "contributors": [
```

Comment:
Was thinking about this, all the other modules are marked as only Passmarked as the author and no contributors. Don't want to get in the game if someone helps to then have to make sure all the names are correctly listed. Thoughts ?